eng/install-native-dependencies.sh

```diff
@@ -11,20 +11,21 @@ set -e
 os="$(echo "$1" | tr "[:upper:]" "[:lower:]")"
 
 if [ -z "$os" ]; then
+    # shellcheck source-path=SCRIPTDIR
     . "$(dirname "$0")"/common/native/init-os-and-arch.sh
 fi
 
 case "$os" in
     linux)
         if [ -e /etc/os-release ]; then
+            # shellcheck source=/dev/null
             . /etc/os-release
         fi
 
         if [ "$ID" = "debian" ] || [ "$ID_LIKE" = "debian" ]; then
             apt update
 
-            apt install -y build-essential gettext locales cmake llvm clang lldb liblldb-dev libunwind8-dev libicu-dev liblttng-ust-dev \
-                libssl-dev libkrb5-dev zlib1g-dev
+            xargs apt-get install -y < eng/debian-reqs.txt
```

Comment:
```suggestion
            apt install -y build-essential gettext locales cmake llvm clang lldb liblldb-dev libunwind8-dev libicu-dev liblttng-ust-dev \
                libssl-dev libkrb5-dev zlib1g-dev
```

lets keep them separate then. I was proposing to deduplicate it the other way around. Makes less sense to change it for one distro leaving others on the table.

Reply:
Could you elaborate? Ubuntu is the only Linux dev platform we "officially" support developing on. Wouldn't it make sense to go through more work to make it a good "reference" platform?

Reply:
This script is idempotent in nature; as it just calls package managers of that nature (worst case: `print "package is already installed"; exit 0`). It supports multiple platforms and has room for more. In the context of this script, there is no official precedence. Currently, it is providing support for Ubuntu, Alpine and Apple platforms.

The suggestion above was to just point people to the usage of this self-documenting script regardless of the distro. If/when someone will hit `Unsupported distro. distro: $ID` branch, they will likely get encouraged to add support for their platform.

Unless we think listing the dependencies for Debian/Ubuntu (without further description) separately is meaningful, IMO, it's better to keep it simple. On the other hand, seeing that we have `eng/Brewfile` also separated (as opposed to `brew install <the whole grocery list>`), we can probably live with this separate file as well. 😅

If we are going to use the separate version, please make sure to use absolute path (so it can be called from any location as before): https://github.com/agocke/runtime/compare/automated-workflow...am11:runtime:patch-35.

Reply:
Yeah brewfile was what I was copying in spirit.